packages/bbui/src/Actions/position_dropdown.js

```diff
@@ -47,7 +48,7 @@ export default function positionDropdown(element, opts) {
         styles.top = anchorBounds.top - elementBounds.height - offset
         styles.maxHeight = maxHeight || 240
       } else {
-        styles.top = anchorBounds.bottom + offset
+        styles.top = anchorBounds.bottom + (offsetBelow || offset)
```

Comment:
Could we not just use the existing `offset` property here?

Reply:
I did try that initially, but unfortunately it didn't work out because the offset is applied when the popover is above the field, or below the field.
In this case I wanted to use the standard offset when above the field, but a large offset when below the field (because of the search input), as shown in the screenshots.